src/main/java/com/thealgorithms/maths/Average.java

```diff
@@ -25,15 +31,15 @@ public static double average(double[] numbers) {
     }
 
     /**
-     * find average value of an int array
+     * Computes the average of an {@code int} array.
      *
-     * @param numbers the array contains element and the sum does not excess long
-     *                value limit
-     * @return average value
+     * @param numbers an array of {@code int} values
+     * @return the average of the given numbers
+     * @throws IllegalArgumentException if the input array is {@code null} or empty
      */
     public static int average(int[] numbers) {
         if (numbers == null || numbers.length == 0) {
-            throw new IllegalArgumentException("Numbers array cannot be empty or null");
```

Comment:
You can keep this line. Do not need to change

Reply:
Got it! I'll keep it as is. 👍